device-connectors/src/testflinger_device_connectors/devices/zapper_kvm/README.md

```diff
@@ -46,3 +48,13 @@ The tool will select the storage device with the following priority:
 ### Noble
 
 Ubuntu OEM 24.04 uses `autoinstall`. The procedure and the arguments are the same as _vanilla_ Ubuntu.
+
+## Live ISO
+
+Support for live ISOs is simply performed booting from an external storage device and returning.
+
+### Job parameters
+
+- __boot_from_ext_media__: should be set to "true"
```

Comment:
Mh maybe the name is confusing or the explanation could be better. If true, it just boots from USB, without any sort of installation. In practice, the Zapper flashed the USB stick, boots from there and returns. Any suggestion to make it clearer?

Reply:
For context, it will be used for the _Ubuntu installer_ testing. Starting point will be either the GRUB screen or the Ubuntu live session.

Reply:
I think I understand, let me see if I have this right. If this is set to true, then it's a live USB that is intended to run completely from the USB booted media rather than an installer? If so, maybe we could just call it "live_usb" or something like that? (possibly no always usb I guess) Or maybe the option we are really asking for here, is whether or not to execute an installation on the thing we boot?

I think the reason I get stuck on it, is that if  "boot_from_ext_media", it sounds to me like we are booting from some internal media - i.e. a system that has already been installed. But really, it sounds like what setting it to fals means is: "still boot from the external media, but then install it".

Reply:
It's a live USB yes, it doesn't run the installer at all because that would be part of the `test_phase` in some scenario. What about `live_iso`, or `live_image`?

Reply:
That sounds more clear to me if it still seems consistent for you?